jest.config.js

```diff
@@ -1,13 +1,30 @@
 module.exports = {
-  testPathIgnorePatterns: ['/node_modules', '<rootDir>/plopfile.js', '/lib/', '/dist/', '/docs/'],
```

Comment:
it looks like `testPathIgnorePatterns` wasn't carried over to the new `jest.config.js` or `jest.preset.js` file

Reply:
I haven't had any issues with it so far. I'll add it back in to see if there is an effect